src/api/system_api.js

```diff
@@ -274,6 +275,7 @@ module.exports = {
             auth: {
                 account: false,
                 system: false,
+                anonymous: true,
```

Comment:
Currently this method is unused, so remove the anonymous. Will update auth when needed.